### DISCUSSION TITLE
Unable To Unpack (Windows)

### DISCUSSION THREAD

**Discussion by fr3dr** [mason-org/mason.nvim#407] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/46770388/189549148-2c20a6ea-04c2-428b-8eff-6838e7ad571e.png)
I'm getting this error and i don't know why, i have git, tar and 7-zip installed.

Thanks in advance!

**Comment by 6mai** [comment, marked as answer]
I had the same error and the problem seems to be that powershell can't use any of the installed archiver software because the environment variables for them haven't been set. Adding 7zip install folder to Path in Advanced system settings > Environment Variables did the trick for me.

**Reply by fr3dr** [reply]
Thanks!
